ntbs-service/wwwroot/css/buttons.scss

```diff
@@ -5,8 +5,7 @@
     box-shadow: 0 4px 0 $button-drop-shadow;
     margin-bottom: 24px;
 
-    &:hover,
-    &:focus {
+    &:hover {
```

Comment:
have they gotten rid of the focus interactions we were copying or are we getting them for free now?

Reply:
Yeah, they made the focus state quite a bit stronger (see https://nhsuk.github.io/nhsuk-frontend/components/button/index.html) now so I thought it's worth us riding off of it, both for the accessibility increase as well as consistency!